plugins/inputs/kinesis_consumer/kinesis_consumer_test.go

```diff
@@ -0,0 +1,140 @@
+package kinesis_consumer
+
+import (
+	"encoding/base64"
+	"github.com/aws/aws-sdk-go/aws"
+	consumer "github.com/harlow/kinesis-consumer"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/parsers"
+	"github.com/influxdata/telegraf/plugins/parsers/json"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type TestTrackingAccumulator struct {
+	telegraf.TrackingAccumulator
+	Metrics *[]telegraf.Metric
+}
+
+func (t TestTrackingAccumulator) AddTrackingMetricGroup(group []telegraf.Metric) telegraf.TrackingID {
+	*t.Metrics = append(*t.Metrics, group...)
+	return 1
+}
```

Comment:
Please use the tracking accumulator provided by [`testutil`](https://github.com/influxdata/telegraf/blob/8a6907a18625a760e5f0f3d95016c6198c83b76b/testutil/accumulator.go#L200).